kernel/api/src/main/java/org/sakaiproject/messaging/api/model/PushSubscription.java

```diff
@@ -60,7 +61,8 @@ public class PushSubscription implements PersistableEntity<Long> {
     @Column(name = "AUTH", length = 255, nullable = false)
     private String auth;
 
-    @Column(name = "ENDPOINT", length = 255, nullable = false)
+    @Lob
+    @Column(name = "ENDPOINT", nullable = false)
```

Comment:
what if we just change length to 2048?

Reply:
I think you get the same thing, a longtext. I'll try it now.

Reply:
No, you're right. You get a varchar(2048). That'll do.

Reply:
I think I'm still stuck on mysql 5.0.2 ...

Reply:
v true maxlength=255 is embedded in my brain for varchar